server/game/cards/04.2-NTB/OlFashionedHangin.js

```diff
@@ -0,0 +1,29 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class OlFashionedHangin extends ActionCard {
+    setupCardAbilities() {
+        this.job({
+            title: 'Ol\' Fashioned Hangin\'',
+            playType: 'noon',
+            target: {
+                activePromptTitle: 'Choose a wanted dude to hang',
+                cardCondition: { location: 'play area', wanted: true, condition: card => card.bounty >= 2 },
+                cardType: ['dude']
+            },
+            message: context => 
+                this.game.addMessage('{0} uses {1} to lead a job to hang {2}', context.player, this, context.target),
+            onSuccess: (job, context) => {
+                this.game.resolveGameAction(GameActions.aceCard({ card: job.mark }), context);
+                if(job.leader.location === 'play area') {
+                    this.game.addMessage('{0} gains 1 CP', job.leader);
```

Comment:
I would always start with the player and put more info here, for example:
`{0}\'s lead executioner {1} gains 1 CP as result of {2}`